src/page/order_pcbs.jsx

```diff
@@ -36,6 +39,19 @@ let OrderPcbs = React.createClass({
               <semantic.Menu.Item as="a" href={aislerUrl}>
                 <img src="/images/aisler.png" />
               </semantic.Menu.Item>
+
+              <semantic.Menu.Item
+                as="a"
+                href={pcbwayUrlPriceComp}
+                onClick={() => {
+                  if (!this.state.downloaded) {
+                    window.open(zipPath)
+                  }
+                }}
+              >
+                
```

Comment:
Where is the image?

Reply:
Also, could this link be third, and differentiated somehow since it's not a PCB service like the others but a comparison.